tendrl/common/utils/service.py

```diff
@@ -0,0 +1,63 @@
+from ansible_module_runner import AnsibleExecutableGenerationFailed
+from ansible_module_runner import AnsibleRunner
+import logging
+
+ANSIBLE_MODULE_PATH = "core/system/systemd.py"
```

Comment:
This supports only systemd services only right?
If we want to support old versions of LINUX where systemd is not available(ex;RHEL6), this would not be enough. I think  the service module(https://github.com/ansible/ansible-modules-core/blob/devel/system/service.py) has support for different init modules including systemd and BSD init. Have considered that?